modules/acl-controller/main.tf

```diff
@@ -0,0 +1,178 @@
+resource "aws_secretsmanager_secret" "client_token" {
+  name = "${var.secret_name_prefix}-consul-client-token"
+}
+
+resource "aws_secretsmanager_secret_version" "client_token" {
+  secret_id     = aws_secretsmanager_secret.client_token.id
+  secret_string = jsonencode({})
+}
+
+resource "aws_ecs_service" "consul-controller" {
+  name            = "consul-acl-controller"
```

Comment:
Should we support the name prefix?

Reply:
actually I just realized I've spent the last 2 years trying to remove prefixes from k8s, nvm 😂

Reply:
I think we should, but only for some resources. Currently none of them are prefixed, and I ran into some problems with that the other day. There are two types or resources for which we'd need a prefix:
1. IAM policies and roles are global. If we don't prefix them we could potentially overwrite a role of another controller running in a different cluster. So those need to be prefixed.
2. Task definitions are shared as well, meaning they don't belong to any specific cluster. So we'd need to prefix the task definition. 

Q: Should we also prefix the service for consistency? I think it'd be kind of nice if it showed just "consul-acl-controller" in the UI without the prefix. The downside is that when you look at tasks, you'll see that the task def is prefixed, and so there will be a disconnect there perhaps.

Reply:
> Should we also prefix the service for consistency?

Either way. I lean slightly toward yes.

Since we are adding a name prefix, can we use it for the secret name prefix too and remove the `secret_name_prefix` variable? The name prefix should be unique enough, since it's used to name globally unique IAM resources.